### PR TITLE
Data connector description update

### PR DESCRIPTION
**Pull request by v-prasadboke**
Required items, please complete
   
   Change(s):
    Data Connector Description Change for the following Solutions 
 - CrowdStrike Falcon
 - Netskope
 - Pulse Connect Secure
 - Qualys VM Knowledge Database

### CODE REVIEW

**Comment by v-rbajaj** [comment]
Hi @v-prasadboke, please fix validations and discard createui changes

**Comment by v-atulyadav** [comment]
Hi @v-rbajaj, please provide your feedback. Thanks

**Comment by v-atulyadav** [comment]
Hi @v-rbajaj, please provide your feedback. Thanks

**Comment by v-rbajaj** [comment]
> Hi @v-rbajaj, please provide your feedback. Thanks

As mentioned earlier, the validations are still failing. 

@v-prasadboke, please fix the validations

File: Solutions/Qualys VM Knowledgebase/Package/mainTemplate.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "_dataConnectorContentId1": "[variables('dataConnectorContentId1')]",
     "dataConnectorId1": "[extensionResourceId(resourceId('Microsoft.OperationalInsights/workspaces', parameters('workspace')), 'Microsoft.SecurityInsights/dataConnectors', variables('_dataConnectorContentId1'))]",
     "_dataConnectorId1": "[variables('dataConnectorId1')]",
-    "dataConnectorTemplateSpecName1": "[concat(parameters('workspace'),'-DataConnector-',variables('_dataConnectorContentId1'))]",
+    "dataConnectorTemplateSpecName1": "[concat(parameters('workspace'),'-dc-',uniquestring(variables('_dataConnectorContentId1')))]",
```
Comment (v-rbajaj) [inline Solutions/Qualys VM Knowledgebase/Package/mainTemplate.json:44]: Hi @v-prasadboke, please revert this change

File: Solutions/Qualys VM Knowledgebase/Package/mainTemplate.json [hunk 2]
```diff
@@ -293,7 +294,7 @@
       "kind": "GenericUI",
       "properties": {
         "connectorUiConfig": {
-          "title": "Qualys VM KnowledgeBase (using Azure Function)",
+          "title": "Qualys VM KnowledgeBase (using Azure Function) (using Azure Function)",
```
Comment (v-rbajaj) [inline Solutions/Qualys VM Knowledgebase/Package/mainTemplate.json:297]: Please remove "(using Azure Function)"  which is twice.

**Comment by v-prasadboke** [comment]
Hello @v-rbajaj any updates on the following`

**Review by v-rbajaj — APPROVED** [review]
Looks fine
